src/Author.java

```diff
@@ -1,6 +1,8 @@
+import java.util.Objects;
+
 public class Author {
-    private String firstname;
-    private String lastname;
+    String firstname;
+    String lastname;
```

Comment:
```suggestion
    String firstName;
    String lastName;
```
названия должны быть в camelCase